objects.js

```diff
@@ -288,7 +315,18 @@ function changeColorOfPlant(plant) {
  * Just watch out for flawed plants!  Don't attempt to clone flawed plants. 
  * Otherwise you will produce flowerless roses.
  */
+
 function cloneAllTheRosesAndChangeTheirColors(estate) {
+    let clonedRoses = [];
+    let newRoses = [];
+    let estateRoses = estate.roseGarden;
```

Comment:
the key name is `roseArbor` make sure you are using console.log() or the debugger to check what your objects look like so you know what values to access and if you are working with the data you think you're working with.